lib/cli.ex

```diff
@@ -0,0 +1,21 @@
+defmodule Cli do
+  def get_integer, do: get_integer("")
+  def get_integer(request) do
+    String.trim(IO.gets request)
```

Comment:
Why oh why isn't this:
request
|> IO.gets
|> String.trim
|> ...

Mixing and matching looks off to me... asymmetrical!